### PR TITLE
 Add support for discord channel

### PR DESCRIPTION
**Pull request by soniasingla**
Added support for discord channel in contribution guidelines :)

### CODE REVIEW

**Review by faith — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Welcome! This project is created by the team at [Embark Studios](https://embark.
 
 At Embark, we aspire to empower everyone to create interactive experiences. To do this, we're exploring and pushing the boundaries of new technologies, and sharing our learnings with the open source community.
 
+If you have any difficulties getting involved or finding answers to your questions, please don't hesitate to ask your questions in our [Discord server](https://discord.com/invite/8TW9nfF).
```
Comment (termhn) [inline CONTRIBUTING.md:7]:
```suggestion
If you have any difficulties getting involved or finding answers to your questions, please don't hesitate to ask in our [Discord server](https://discord.com/invite/8TW9nfF).
```

**Review by termhn — CHANGES_REQUESTED** [review]
minor grammar nit

**Comment by khyperia** [comment]
The discord is already linked in CONTRIBUTING.md, could we clean that up and not link it twice?

**Comment by faith** [comment]
> The discord is already linked in CONTRIBUTING.md, could we clean that up and not link it twice?

Its not only linked to contributing, discord can be used for other stuff too, so why not.

**Comment by khyperia** [comment]
> Its not only linked to contributing, discord can be used for other stuff too, so why not.

I think there's a misunderstanding here. This PR is adding a link to the discord in CONTRIBUTING.md, but there is already a link to the discord in CONTRIBUTING.md, and it'd be nice if there was only one link. I'm not sure how discord being able to be used for things other than contributing is relevant here.

**Comment by faith** [comment]
> > Its not only linked to contributing, discord can be used for other stuff too, so why not.
> 
> I think there's a misunderstanding here. This PR is adding a link to the discord in CONTRIBUTING.md, but there is already a link to the discord in CONTRIBUTING.md, and it'd be nice if there was only one link. I'm not sure how discord being able to be used for things other than contributing is relevant here.

Ah nevermind

**Pull request closed without merging**